src/services/documentService.ts

```diff
@@ -37,6 +40,36 @@ export default class DocumentService {
     });
   }
 
+  public async signDocument(
+    unsignedContentBase64: string
+  ): Promise<Either<Error, string>> {
+    try {
+      const soapClient: soap.Client = await soap.createClientAsync(
+        getRequiredEnvVar("ARSS_WSDL_URL")
+      );
+      const result = await soapClient.pdfsignatureV2Async({
+        SignRequestV2: {
+          certID: "AS0",
```

Comment:
what's `AS0` here ? (may we have a comment that links to the service documentation here ?)